test/e2e/conditions.go

```diff
@@ -0,0 +1,61 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/types"
+	yaml "sigs.k8s.io/yaml"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+type ExpectedConditionsStatus struct {
+	Node       string
+	conditions nmstatev1alpha1.ConditionList
+}
+
+func conditionsToYaml(conditions nmstatev1alpha1.ConditionList) string {
+	manifest, err := yaml.Marshal(conditions)
+	if err != nil {
+		panic(err)
+	}
+	return string(manifest)
+}
+func enactmentConditionsStatus(node string) nmstatev1alpha1.ConditionList {
```

Comment:
Please insert an empty line between functions. Is not that handled by gofmt?

Reply:
Is not handle by it.